google-cloud-tools-plugin/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'org.jetbrains.intellij'
 intellij {
 
     pluginName = 'google-cloud-tools'
-    plugins 'Groovy', 'gradle', 'git4idea', 'properties', 'junit', 'maven', 'yaml'
+    plugins 'Groovy', 'git4idea', 'gradle', 'properties', 'junit', 'maven', 'yaml'
```

Comment:
I wonder is it just ordering or changing their order does change something in the build process?